### PR TITLE
moving accordion out to mofo-ui

### PR DESCRIPTION
**Pull request by gvn**
Requires: https://github.com/mozilla/mofo-ui/pull/7 and a bump of mofo-ui before landing.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ module.exports = {
   entry: {
     javascript: `./react/main.jsx`
   },
+  resolve: {
+    alias: {
+      // Prevent multiple copies of React from being loaded when mofo-ui is used
```
Comment (gvn) [inline webpack.config.js]: It's baaaaaack. `rc-collapse` being a sub-dependency seems to necessitate this.

Comment (alanmoo) [inline webpack.config.js]: 😖🤔

**Comment by alanmoo** [comment]
As we discussed in Chat, a recent change results in an development version of react warning in the console. This is likely due to the way mofo-ui is distributed, so let's hold off on this PR until that's fixed.

**Comment by gvn** [comment]
Ok, give this a shot with this PR npm link'd:

https://github.com/mozilla/mofo-ui/pull/11

**Comment by gvn** [comment]
This should work now. However, before landing we need to land https://github.com/mozilla/mofo-ui/pull/16 and do another bump of `mofo-ui`. 

Right now there's some conflicting CSS because the tabswitcher in Science isn't the same as the one in mofo-ui. It probably will be eventually, but for now a namespace solves the problem...

**Comment by gvn** [comment]
Ok, everything should be working now.

File: webpack.config.js [hunk 2]
```diff
@@ -17,6 +17,12 @@ module.exports = {
       }
     })
   ],
+  resolve: {
```
Comment (alanmoo) [inline webpack.config.js]: (Why) is this still necessary?

**Comment by alanmoo** [comment]
works for me, just curious about that webpack config- I'd hope to not need that in future projects just because we're using mofo-ui.

File: webpack.config.js (hunk 2 above)
Comment (gvn) [inline webpack.config.js]: Hmm...I guess maybe it's not now that I'm not `npm link`ing. Seems to work without.

**Pull request merged**